app/components/avo/panel_component.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
 class Avo::PanelComponent < ViewComponent::Base
+  include Avo::ApplicationHelper
+
   attr_reader :title # deprecating title in favor of name
   attr_reader :name
   attr_reader :classes
+  attr_reader :reload_button
```

Comment:
We used to add `attr_reader` on our components over using the instance variable directly (`@reload_button`) 

I did some benchmark and using directly the instance variable is always faster.

Let's don't use `attr_reader` anymore unless there is a need to access the variable outside the component like `Avo::PanelComponent.new.reload_button`

```suggestion
```

Reply:
Ok, I will change it.